vscode-trace-extension/src/extension.ts

```diff
@@ -6,8 +6,12 @@ import { TraceExplorerAvailableViewsProvider } from './trace-explorer/available-
 import { TraceExplorerOpenedTracesViewProvider } from './trace-explorer/opened-traces/trace-explorer-opened-traces-webview-provider';
 import { fileHandler, openOverviewHandler, resetZoomHandler } from './trace-explorer/trace-tree';
 import { updateTspClient } from './utils/tspClient';
+import { TraceExtensionLogger } from './utils/trace-extension-logger';
+
+export let traceLogger: TraceExtensionLogger;
 
 export function activate(context: vscode.ExtensionContext): void {
+    traceLogger = new TraceExtensionLogger('Trace Extension');
```

Comment:
I'm unable to locate this option in the `Developer: Set Log Level` menu of the executing extension (from the command palette), but maybe I'm missing something simply enough. Can you summarize the steps leading to enabling these logs in the Output panel, maybe in the root `README.md`? -With a typical usage example if possible.